examples/blueprint-construct/index.ts

```diff
@@ -5,6 +5,7 @@ import { Construct } from "constructs";
 import * as blueprints from '../../lib';
```

Comment:
Fix the GitHub actions warnings.

Reply:
I am not sure what you mean by that? Are you saying I should have it instead of ../../lib be the `import * as blueprints from '@aws-quickstart/eks-blueprints'`?

Reply:
Use file view : https://github.com/aws-quickstart/cdk-eks-blueprints/pull/440/files#diff-40d4a72c979379c5b6ab47139c410af18bd607fd5b4f8a26d1aaef9956d8d447

scroll down from my comment and observe github actions warnings.